torch/csrc/lazy/ts_backend/ops/device_data.h

```diff
@@ -12,14 +12,26 @@ class TORCH_API DeviceData : public TsNode {
 
   explicit DeviceData(std::shared_ptr<BackendData> data);
 
+  bool Equal(std::shared_ptr<BackendData> data) const {
+    return data_->shape() == data->shape();
```

Comment:
If I understand correctly, this equal is being used by the TrieCache hence we only compare the shape(instead of the handle). Can we add a comment to make this all clear?